metadata_backend/api/operators.py

```diff
@@ -660,9 +660,12 @@ async def delete_folder(self, folder_id: str) -> None:
             LOG.error(reason)
             raise web.HTTPBadRequest(reason=reason)
         if not delete_success:
-            reason = "Deleting for {folder_id} from database failed."
+            reason = f"Deleting for {folder_id} from database failed."
             LOG.error(reason)
             raise web.HTTPBadRequest(reason=reason)
+        else:
+            LOG.info(f"Deleting folder with id {folder_id} to database succeeded.")
+            return folder_id
```

Comment:
Ok, so this a pretty good catch and it wouldn't have passed in previous PR, had there been better testing done then, whoops 😅

However, I'm wondering if returning the `folder_id` is that necessary. The `delete_folder()` method in `handlers.py` could instead just be:
```
        folder_id = req.match_info["folderId"]
        db_client = req.app["db_client"]
        operator = FolderOperator(db_client)
        await operator.delete_folder(folder_id)
        LOG.info(f"DELETE folder with ID {folder_id} was successful.")
        return web.Response(status=204)
```

Reply:
yeah this could respond with 204`, did not noticed it at first we should keep it consistent with other delete `endpoints`

Reply:
good point, will fix

Reply:
i think it's better to actually catch result from awaited delete folder operation and then log. Otherwise you're logging that delete was succesful before it was actually executed

Reply:
so actually, I would argue no change is needed here

Reply:
alright, fair enough